src/gwt/src/org/rstudio/studio/client/projects/StudioClientProjectConstants_fr.properties

```diff
@@ -139,6 +139,7 @@ chooseFileCaption=Choisir un fichier
 enableCodeIndexingLabel=Indexer les fichiers sources (pour la recherche/navigation dans le code)
 chkSpacesForTabLabel=Insérer des espaces pour les tabulations
 tabWidthLabel=Largeur de la tabulation
+insertNativePipeOperatorLabel=Utiliser l''opérateur pipe natif de R, |> PLEASE UPDATE
```

Comment:
This translation is missing the parenthetical (Requires R 4.1+) 
Could someone please add?